File: PAT/Basic Level/1006.cpp
#include <iostream>

using std::cin;
using std::cout;

int main()
{
  int number;
  cin >> number;

  int bai = number / 100;
  number %= 100;
  int shi = number / 10;
  number %= 10;

  for(int i = 0; i < bai; i++)
    cout <<'B';
  for(int i = 0; i < shi; i++)
    cout << 'S';
  for(int i = 0; i < number; i++)
    cout << i + 1;

  return 0;
}

File: PAT/Basic Level/1006.py
number = int(raw_input())

bai = number / 100
number %= 100
shi = number / 10
number %= 10
strnum = bai * 'B'
strnum += shi * 'S'
for i in range(0, number):
    strnum += str(i+1)

print strnum
